maquetado/scss/application.scss

```diff
@@ -0,0 +1 @@
+@import 'detail.scss';
```

Comment:
I think you are missing the equalizer. Check this card https://trello.com/c/6LXHgdRU/17-css-resets